### PR TITLE
Add San Francisco

### PR DESCRIPTION
**Pull request by anipra**
This is beginning of San Francisco file. 

Please provide feedback.

- what do you like about San Francisco?
- if you have never been to San Francisco, what interests you ?

### CODE REVIEW

**Comment by n9tq** [comment]
- Love the variety of food
- Would you happen to know anything about scuba diving in the area

File: anipra-walnutcreek.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# San Francisco, CA
+
+### Restaurants
+
+### Local Attractions
+
+- Fisherman's wharf
```
Comment (AllstonS) [inline anipra-walnutcreek.md]: I was there when I was a teenager. I loved it. Are you from San Francisco originally?

**Comment by anipra** [comment]
There are couple of training lessons here for scuba diving. Both open and close water.

File: anipra-walnutcreek.md (hunk 1 above)
Comment (anipra) [inline anipra-walnutcreek.md]: No. I work in San Francisco. It is a beautiful city.

**Comment by Sreeparvathy** [comment]
Ready to 🚢

**Pull request merged**